Src/lib/drivers/button/button.c

```diff
@@ -7,12 +7,14 @@
 #include "button.h";
 #define MIN_PRESSED_TIME_BORDER 0
 #define MAX_PRESSED_TIME_BORDER 10000
+#define DEBOUNCER 50
```

Comment:
Debouncer of what? Time, penguins🙂?
For example you can change it to DEBOUNCER_TIME_MS(S)

Reply:
Done